lib/fathead/htmlref/parse.py

```diff
@@ -20,29 +20,51 @@ def replace_all(text, terms):
 class Tag(object):
     """ Contains informations about
     a HTML tag """
-    def __init__(self, name, info, reference, example):
+    def __init__(self, name, info, reference, example, see_also):
         self.name = name
         self.info = info
         self.reference = reference
-        
-        # Remove excess padding around synopsis
+        self.see_also = see_also
+
+        # Remove excess padding around code snippet
         self.example = re.sub('^\\n', '', example)
         self.example = re.sub('\\n$', '', self.example)
 
-        self.example = replace_all(self.example, {'\n': '\\n',
-                                             '\t': '\\t',
-                                             '\r': ''})
+        # replace all newline and horizontal tab characters
+        terms = {'\n': '\\n', '\t': '\\t', '\r': ''}
+        self.info = cgi.escape(replace_all(self.info, terms))
+        self.example = cgi.escape(replace_all(self.example, terms))
+
+    def make_abstract(self):
+        """ Creates the abstract using
+        the new fathead template elements """
+        abstract = ''
+
+        if self.example:
+            abstract += '<pre><code>' + self.example + '</pre></code>'
```

Comment:
@sriranganathan this is the bug! it should be `</code></pre>`

Once you fix this, you can remove the HTML entity escaping